locales/en/plugin__mcg-ms-console.json

```diff
@@ -94,6 +94,7 @@
   "Data sources": "Data sources",
   "Replication buckets": "Replication buckets",
   "Delete bucket": "Delete bucket",
+  "Data Federation": "Data Federation",
```

Comment:
it is getting added because somewhere we are still passing data federation to the translation. instead, use the constant - DATA_FEDERATION